packages/cli/src/handlers/login.ts

```diff
@@ -107,8 +107,10 @@ export const login = async (url: string, options: LoginOptions) => {
         await setProjectInteractively({ verbose: options.verbose });
     } catch (e: any) {
         if (options.verbose)
-            console.error(`> Set project returned response: ${e.statusCode}`);
-        if (e.statusCode === 404) {
+            console.error(
+                `> Set project returned response: ${JSON.stringify(e)}`,
```

Comment:
Does this solve this issue ? #3323

Reply:
Not exactly. The error is fixed on the backend https://github.com/lightdash/lightdash/pull/3337/files#r983189676 (previously we were returnin an empty list) the rest is error handling